.github/workflows/production.yml

```
@@ -54,6 +54,13 @@ jobs:
     - name: Push with commit ${{ matrix.php }} tag
       run: docker push ghcr.io/${{ env.CI_REPOSITORY_OWNER_SLUG }}/${{ env.CI_REPOSITORY_NAME }}:${{ matrix.php }}
 
+    #review containers
+    - name: Build the review container image
+      run: docker build . --tag ghcr.io/${{ github.repository_owner }}/${{ env.CI_REPOSITORY_NAME }}:${{ matrix.php }}-review --file Dockerfile.${{ matrix.php }}-review
+    - name: Push with commit *-review tag
+      run: docker push ghcr.io/${{ github.repository_owner }}/${{ env.CI_REPOSITORY_NAME }}:${{ matrix.php }}-review
```

```suggestion
      run: docker push ghcr.io/${{ env.CI_REPOSITORY_OWNER_SLUG }}/${{ env.CI_REPOSITORY_NAME }}:${{ matrix.php }}-review
```

Should we use this one instead?
`ghcr.io` [evidently](https://github.com/orgs/community/discussions/27086) only accepts `a-z` owner names. `${{ env.CI_REPOSITORY_OWNER_SLUG }}` is always lowercase, while `${{ github.repository_owner }}` can contain uppercase characters too.

For `linkorbb/linkorb`, it doesn't matter. But I had problems with my forks of this because my GitHub username is `Ayesh`.